parkings/models/permit.py

```diff
@@ -175,14 +175,14 @@ def by_subject(self, registration_number):
         return self.filter(registration_number=normalized_reg_num)
 
     def by_area(self, area_identifier):
-        return self.filter(area_identifier=area_identifier)
+        return self.filter(area__identifier=area_identifier)
```

Comment:
Here too, should either take the domain as an argument like `def by_area(domain, area_identifier)` or an area object `def by_area(area): assert isinstance(area, PermitArea)`.